iceoryx2-cli/iox2-config/src/commands.rs

```diff
@@ -0,0 +1,127 @@
+//  Copyright (c) 2024 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache Software License 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+// which is available at https://opensource.org/licenses/MIT.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use anyhow::Result;
+use enum_iterator::all;
+use iceoryx2::config::Config;
+use iceoryx2_bb_posix::system_configuration::*;
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+
+/// Prints the whole system configuration with all limits, features and details to the console.
+pub fn print_system_configuration() {
```

Comment:
OK for this PR, but there might be a better place to but this. @elfenpiff Thoughts?

Reply:
For now I would keep it here. I also do not yet have an answer where it would fit best ... time will show